artemis-commons/src/main/java/org/apache/activemq/artemis/utils/PasswordMaskingUtil.java

```diff
@@ -54,7 +54,7 @@ private LazySecureProcessorHolder() {
       static {
          HashProcessor processor = null;
          Exception exception = null;
-         final String codecDesc = "org.apache.activemq.artemis.utils.DefaultSensitiveStringCodec;algorithm=one-way";
```

Comment:
previously encoded passwords will still work?

Reply:
Yes.  There's no semantic change here.  Just changing a hard-coded string to something a little more dynamic.

Reply:
@jbertram duh.. I just read the actual change... thanks!